### PR TITLE
add default TTL

### PR DESCRIPTION
**Pull request by AntonPuko**

### CODE REVIEW

File: dist/server/DeviceServer.js [hunk 1]
```diff
@@ -275,7 +275,7 @@ var DeviceServer = function () {
                   deviceID: deviceID,
                   isPublic: isPublic,
                   name: message.getUriPath().substr(3),
-                  ttl: message.getMaxAge() > 0 ? message.getMaxAge() : 60,
+                  ttl: message.getMaxAge() > 0 && message.getMaxAge(),
```
Comment (jlkalberer) [inline dist/server/DeviceServer.js]: I don't think this is correct. You should just replace `0` with the `DEFAULT_EVENT_TTL`

File: src/lib/EventPublisher.js [hunk 2]
```diff
@@ -27,6 +27,7 @@ import nullthrows from 'nullthrows';
 import uuid from 'uuid';
 
 const ALL_EVENTS = '*all*';
+const DEFAULT_EVENT_TTL = 60;
```
Comment (jlkalberer) [inline src/lib/EventPublisher.js]: Maybe this goes in settings so it's shared everywhere.

File: src/server/DeviceServer.js [hunk 3]
```diff
@@ -256,7 +256,7 @@ class DeviceServer {
       deviceID,
       isPublic,
       name: message.getUriPath().substr(3),
-      ttl: message.getMaxAge() > 0 ? message.getMaxAge() : 60,
+      ttl: message.getMaxAge() > 0 && message.getMaxAge(),
```
Comment (jlkalberer) [inline src/server/DeviceServer.js]: Same with this... We should be using the constant if it's zero.

**Review by jlkalberer — CHANGES_REQUESTED** [review]

File: src/settings.js [hunk 4]
```diff
@@ -18,18 +18,8 @@
 *
 */
 
-
-import path from 'path';
-
 export default {
-  DEVICE_DIRECTORY: path.join(__dirname, './data/deviceKeys'),
```
Comment (jlkalberer) [inline src/settings.js]: These actually need to be here.

**Review by jlkalberer — CHANGES_REQUESTED** [review]

File: src/settings.js (hunk 4 above)
Comment (AntonPuko) [reply src/settings.js]: ops, fixed

**Pull request merged**